src/internal/observable/from.ts

```diff
@@ -130,17 +131,16 @@ export function from<T>(input: ObservableInput<T>, scheduler?: SchedulerLike): O
 }
 
 function subscribeTo<T>(result: ObservableInput<T>): (subscriber: Subscriber<T>) => Subscription | void {
-  if (result && typeof (result as any)[Symbol_observable] === 'function') {
+  if (isFunction((result as any)[Symbol_observable])) {
```

Comment:
issue: This isn't quite the same. `result` is checked, so I think optional chaining is needed to effect exactly the same behaviour:
```
isFunction((result as any)?.[Symbol_observable])
```

Reply:
I've opted instead to just check `result != null` at the start. As it would be more efficient.